### ISSUE TITLE
Multiple consecutive ellipsis tokens in input are not preserved

### ISSUE THREAD

**Issue by amir-zeldes** [udapi/udapi-python#73]
If an input sentence contains multiple consecutive ellipsis tokens (with decimal IDs), only the first is preserved in output. For example if we have the following sentence:

```
sent = '''1	She	she	PRON	PRP		2	nsubj	_	_
2	began	begin	VERB	VBD	_	0	root	_	_
3	playing	play	VERB	VBG	_	2	xcomp	_	_
4	small	small	ADJ	JJ	_	5	amod	_	_
5	roles	role	NOUN	NNS		3	obj	_	_
6	in	in	ADP	IN	_	7	case	_	_
7	films	film	NOUN	NNS		3	obl	_	_
8	in	in	ADP	IN	_	9	case	_	_
9	1949	1949	NUM	CD		3	obl	_	SpaceAfter=No
10	,	,	PUNCT	,	_	2	punct	_	_
11	with	with	ADP	IN	_	13	case	_	_
12	impressive	impressive	ADJ	JJ	_	13	amod	_	_
13	performances	performance	NOUN	NNS		3	obl	_	_
14	in	in	ADP	IN	_	17	case	_	_
15	the	the	DET	DT	_	17	det	_	_
16	Fernandel	Fernandel	PROPN	NNP		17	compound	_	_
17	vehicle	vehicle	NOUN	NN		13	nmod	_	_
18	Meurtres	Meurtres	PROPN	NNP		17	appos	_	SpaceAfter=No
19	?	?	PUNCT	.		2	punct	_	_
20	(	(	PUNCT	-LRB-		2	punct	_	SpaceAfter=No
21	Three	Three	NUM	NNP	_	22	nummod	_	_
22	Sinners	Sinners	PROPN	NNP		18	appos	_	SpaceAfter=No
23	,	,	PUNCT	,		2	punct	_	_
24	1950	1950	NUM	CD		22	nmod:tmod	_	SpaceAfter=No
25	)	)	PUNCT	-RRB-		2	punct	_	_
26	and	and	CCONJ	CC	_	28	cc	_	_
26.1	impressive	impressive	ADJ	JJ	Degree=Pos	_	_	26.2:amod	CopyOf=12
26.2	performances	performance	NOUN	NNS	Number=Plur	_	_	13:conj:and	CopyOf=13
27	alongside	alongside	ADP	IN	_	28	case	_	_
28	Jean	Jean	PROPN	NNP		13	conj	26.2:nmod:alongside	_
29	Gabin	Gabin	PROPN	NNP	_	28	flat	_	_
30	as	as	ADP	IN	_	32	case	_	_
31	a	a	DET	DT	_	32	det	_	_
32	showgirl	showgirl	NOUN	NN		28	orphan	26.2:nmod:as	_
33	/	/	SYM	SYM	_	36	cc	_	_
34	gangster	gangster	NOUN	NN		36	nmod	_	SpaceAfter=No
35	's	's	PART	POS	_	34	case	_	SpaceAfter=No
36	moll	moll	NOUN	NN	_	32	conj	26.2:nmod:as|32:conj:slash	_
37	in	in	ADP	IN	_	39	case	_	_
38	the	the	DET	DT	_	39	det	_	_
39	film	film	NOUN	NN		28	orphan	26.2:nmod:in	_
40	Touchez	Touchez	PROPN	NNP		39	appos	_	_
41	pas	pas	PROPN	NNP	_	40	flat	_	_
42	au	au	PROPN	NNP	_	40	flat	_	_
43	grisbi	grisbi	PROPN	NNP	_	40	flat	_	_
44	(	(	PUNCT	-LRB-		2	punct	_	SpaceAfter=No
45	1954	1954	NUM	CD		40	nmod:tmod	_	SpaceAfter=No
46	)	)	PUNCT	-RRB-		2	punct	_	SpaceAfter=No
47	.	.	PUNCT	.	_	2	punct	_	_'''
```

The token 26.2 will be destroyed in output when running:

```
from udapi.core.document import Document
doc = Document()
doc.from_conllu_string(sent)
print(doc.to_conllu_string())
```

And the unexpected output missing the token 26.2 is:

```
...
26	and	and	CCONJ	CC	_	28	cc	_	Discourse=same-unit:35->33
26.1	impressive	impressive	ADJ	JJ	Degree=Pos	_	_	26.2:amod	CopyOf=12
27	alongside	alongside	ADP	IN	_	28	case	_	_
...
```

**Comment by martinpopel** [comment]
Can you try this with the newest version from GitHub? It works for me. I've done several updates in the recent days, including empty nodes.

BTW: You can try `print(doc)` or e.g. `doc.draw(layout='align',color=1,attributes='form,upos,deprel,misc')` and you will see a rendering including the empty nodes.
```
# sent_id = 1
# text = She began playing small roles in films in 1949, with impressive performances in the Fernandel vehicle Meurtres? (Three Sinners, 1950) and alongside Jean Gabin as a showgirl / gangster'smoll in the film Touchez pas au grisbi (1954).
─┮                                               
 │ ╭─╼               She          PRON  nsubj     _
 ╰─┾                 began        VERB  root      _
   ┡───┮             playing      VERB  xcomp     _
   │   │ ╭─╼         small        ADJ   amod      _
   │   ┡─┶           roles        NOUN  obj       _
   │   │ ╭─╼         in           ADP   case      _
   │   ┡─┶           films        NOUN  obl       _
   │   │ ╭─╼         in           ADP   case      _
   │   ┡─┶           1949         NUM   obl       SpaceAfter=No
   ┡─╼ │             ,            PUNCT punct     _
   │   │ ╭─╼         with         ADP   case      _
   │   │ ┢─╼         impressive   ADJ   amod      _
   │   ╰─┾           performances NOUN  obl       _
   │     │ ╭─╼       in           ADP   case      _
   │     │ ┢─╼       the          DET   det       _
   │     │ ┢─╼       Fernandel    PROPN compound  _
   │     ┡─┾         vehicle      NOUN  nmod      _
   │     │ ╰─┮       Meurtres     PROPN appos     SpaceAfter=No
   ┡─╼   │   │       ?            PUNCT punct     _
   ┡─╼   │   │       (            PUNCT punct     SpaceAfter=No
   │     │   │ ╭─╼   Three        NUM   nummod    _
   │     │   ╰─┾     Sinners      PROPN appos     SpaceAfter=No
   ┡─╼   │     │     ,            PUNCT punct     _
   │     │     ╰─╼   1950         NUM   nmod:tmod SpaceAfter=No
   ┡─╼   │           )            PUNCT punct     _
   │     │ ╭─╼       and          CCONJ cc        _
   │     │ │         impressive   ADJ   _         CopyOf=12
   │     │ │         performances NOUN  _         CopyOf=13
   │     │ ┢─╼       alongside    ADP   case      _
   │     ╰─┾         Jean         PROPN conj      _
   │       ┡─╼       Gabin        PROPN flat      _
   │       │ ╭─╼     as           ADP   case      _
   │       │ ┢─╼     a            DET   det       _
   │       ┡─┾       showgirl     NOUN  orphan    _
   │       │ │ ╭─╼   /            SYM   cc        _
   │       │ │ ┢─┮   gangster     NOUN  nmod      SpaceAfter=No
   │       │ │ │ ╰─╼ 's           PART  case      SpaceAfter=No
   │       │ ╰─┶     moll         NOUN  conj      _
   │       │ ╭─╼     in           ADP   case      _
   │       │ ┢─╼     the          DET   det       _
   │       ╰─┾       film         NOUN  orphan    _
   │         ╰─┮     Touchez      PROPN appos     _
   │           ┡─╼   pas          PROPN flat      _
   │           ┡─╼   au           PROPN flat      _
   │           ┡─╼   grisbi       PROPN flat      _
   ┡─╼         │     (            PUNCT punct     SpaceAfter=No
   │           ╰─╼   1954         NUM   nmod:tmod SpaceAfter=No
   ┡─╼               )            PUNCT punct     SpaceAfter=No
   ╰─╼               .            PUNCT punct     _
```

**Comment by martinpopel** [comment]
BTW: you can fix the errors in punctuation attachment using:
```python
import udapi
doc = udapi.Document('my.conllu')
udapi.create_block('ud.FixPunct').apply_on_document(doc)
doc.draw(layout='align',color=1,attributes='form,upos,deprel,misc')
```

```
# sent_id = 1
# text = She began playing small roles in films in 1949, with impressive performances in the Fernandel vehicle Meurtres? (Three Sinners, 1950) and alongside Jean Gabin as a showgirl / gangster'smoll in the film Touchez pas au grisbi (1954).
─┮                                             
 │ ╭─╼             She          PRON  nsubj     _
 ╰─┾               began        VERB  root      _
   ┡─┮             playing      VERB  xcomp     _
   │ │ ╭─╼         small        ADJ   amod      _
   │ ┡─┶           roles        NOUN  obj       _
   │ │ ╭─╼         in           ADP   case      _
   │ ┡─┶           films        NOUN  obl       _
   │ │ ╭─╼         in           ADP   case      _
   │ ┡─┶           1949         NUM   obl       SpaceAfter=No
   │ │ ╭─╼         ,            PUNCT punct     _
   │ │ ┢─╼         with         ADP   case      _
   │ │ ┢─╼         impressive   ADJ   amod      _
   │ ╰─┾           performances NOUN  obl       _
   │   │ ╭─╼       in           ADP   case      _
   │   │ ┢─╼       the          DET   det       _
   │   │ ┢─╼       Fernandel    PROPN compound  _
   │   ┡─┾         vehicle      NOUN  nmod      _
   │   │ ╰─┮       Meurtres     PROPN appos     SpaceAfter=No
   │   │   ┡─╼     ?            PUNCT punct     _
   │   │   │ ╭─╼   (            PUNCT punct     SpaceAfter=No
   │   │   │ ┢─╼   Three        NUM   nummod    _
   │   │   ╰─┾     Sinners      PROPN appos     SpaceAfter=No
   │   │     │ ╭─╼ ,            PUNCT punct     _
   │   │     ┡─┶   1950         NUM   nmod:tmod SpaceAfter=No
   │   │     ╰─╼   )            PUNCT punct     _
   │   │ ╭─╼       and          CCONJ cc        _
   │   │ │         impressive   ADJ   _         CopyOf=12
   │   │ │         performances NOUN  _         CopyOf=13
   │   │ ┢─╼       alongside    ADP   case      _
   │   ╰─┾         Jean         PROPN conj      _
   │     ┡─╼       Gabin        PROPN flat      _
   │     │ ╭─╼     as           ADP   case      _
   │     │ ┢─╼     a            DET   det       _
   │     ┡─┾       showgirl     NOUN  orphan    _
   │     │ │ ╭─╼   /            SYM   cc        _
   │     │ │ ┢─┮   gangster     NOUN  nmod      SpaceAfter=No
   │     │ │ │ ╰─╼ 's           PART  case      SpaceAfter=No
   │     │ ╰─┶     moll         NOUN  conj      _
   │     │ ╭─╼     in           ADP   case      _
   │     │ ┢─╼     the          DET   det       _
   │     ╰─┾       film         NOUN  orphan    _
   │       ╰─┮     Touchez      PROPN appos     _
   │         ┡─╼   pas          PROPN flat      _
   │         ┡─╼   au           PROPN flat      _
   │         ┡─╼   grisbi       PROPN flat      _
   │         │ ╭─╼ (            PUNCT punct     SpaceAfter=No
   │         ╰─┾   1954         NUM   nmod:tmod SpaceAfter=No
   │           ╰─╼ )            PUNCT punct     SpaceAfter=No
   ╰─╼             .            PUNCT punct     _
```

**Comment by martinpopel** [comment]
OK, [released udapi 0.2.3](https://pypi.org/project/udapi/) and closing this issue.

**Comment by amir-zeldes** [comment]
Great, thanks!
